### PR TITLE
FS: "saving..." indicator

### PR DESCRIPTION
**Pull request by alexyoon97**
### What this PR does (required):
- indicating whether the data is on saving or saved

### Screenshots / Videos (required):
https://user-images.githubusercontent.com/43189680/128811324-4c6bb220-ac46-44bb-ad20-1b06fbdf978d.mp4

### Any information needed to test this feature (required):
- Implement SaveIndicator component where this feature should be added.
- SaveIndicator component will accept one boolean prop to trigger saving to saved

### Any issues with the current functionality (optional):
- Maybe a little better font design would be a future improvement

### CODE REVIEW

**Review by ajaytomgeorge — COMMENTED** [review]
@moffatethan Looks good from my side

File: client/src/components/SaveIndicator/SaveIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  bStatus: boolean;
+}
+
+const SaveIndicator = ({ bStatus }: SIProps): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <>
+      {
+        <div className={classes.saving_container}>
+          {bStatus ? (
```
Comment (moffatethan) [inline client/src/components/SaveIndicator/SaveIndicator.tsx:15]: bStatus? boardStatus, might be a bit more descriptive to just put `boardStatus`

**Review by moffatethan — APPROVED** [review]
Left a comment to address, otherwise, this is good

File: client/src/components/SaveIndicator/SaveIndicator.tsx (hunk 1 above)
Comment (alexyoon97) [reply client/src/components/SaveIndicator/SaveIndicator.tsx:15]: I was meaning boolean status to let others know its boolean value, but I can make a change if that's more readable.

**Pull request merged**